sass/directives/_tables.scss

```diff
@@ -38,76 +38,146 @@
   border-top-right-radius: $table-header-corner-radius;
 }
 
-@mixin table--product($table-highlight-z-index: 1, $table-caption-z-index: 2, $table-col-width: 25%) {
-  @include table--row-column-headers($table-col-width, $table-caption-z-index);
-  overflow: hidden;
+
+// Compare Charts
+
+@mixin table--row-column-headers($table-caption-z-index: 2) {
+  border-collapse: collapse;
+  border-spacing: $table-border-spacing;
+  table-layout: fixed;
+  width: 100%;
+
+  caption {
+    position: relative;
+    z-index: $table-caption-z-index;
+    background: $panel-primary;
+    text-align: left;
+  }
 
   th, td {
-    // Column highlight
-    @include table__column-highlight('', $table-highlight-z-index);
+    border: $base-border;
   }
 
-  // Remove column highlight
-  th[scope='row'],
-  thead td {
-    @include table__column-highlight(none, $table-highlight-z-index);
+  th {
+    font-weight: $font-weight-bold;
+    text-transform: uppercase;
+    font-size: 0.9375rem;
   }
 
-  thead {
-    th {
-      color: white;
-      font-weight: $font-weight-bold;
-    }
+  thead th:first-of-type {
+    border: 0;
+    background-color: transparent;
 
-    th:nth-of-type(1n) {
-      background-color: $table-column-heading-1;
+    @media only screen and (max-width: $medium-screen-max) {
+      width: 14ch;
     }
+  }
 
-    th:nth-of-type(2n) {
-      background-color: $table-column-heading-2;
-    }
+  th[scope='col'] {
+    padding: $base-spacing-medium;
+    color: white;
+    text-align: center;
+    position: relative;
+    background-clip: padding-box;
+  }
 
-    th:nth-of-type(3n) {
-      background-color: $table-column-heading-3;
+  th[scope='row'] {
+    padding: $table--row-column-headers--row-top-bottom-padding $base-spacing-medium;
```

Comment:
I could see using `table--row-column-headers` as the name for that type of mixin and `row-column-headers-table--row-top-bottom-padding` or `row-column-headers-table__row-top-bottom-padding` as the name for this variable to remove one level of naming